.github/workflows/image.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           # If the VERSION looks like a version number, assume that
           # this is the most recent version of the image and also
           # tag it 'latest'.
-          if [[ $VERSION =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
+          if [[ $VERSION =~ ^v[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
```

Comment:
because `v` is part of what `git describe` returns:

```
~/workspace/kairos/AuroraBoot (also-push-latest)*$ git describe --always --tags --dirty
v0.3.2-1-g2fea056
```